photon-api/src/main/scala/com/linkedin/photon/ml/supervised/classification/SmoothedHingeLossLinearSVMModel.scala

```diff
@@ -33,35 +34,91 @@ class SmoothedHingeLossLinearSVMModel(override val coefficients: Coefficients)
   with Serializable {
 
   /**
-   * Compute the mean of the soft hinge loss linear SVM model
+   *
+   * @return The model type.
+   */
+  override def modelType: TaskType = SMOOTHED_HINGE_LOSS_LINEAR_SVM
```

Comment:
Just a general comment: this is duplicating type information that is already present in the class definition. Usually this is frowned upon in other OOLs.